docs/apis.md

```diff
@@ -7,8 +7,19 @@ SPDX-License-Identifier: Apache-2.0
 
 ## gNBsim can Optionally launch profiles through HTTP APIs
 
-gNBSim can process HTTP Requests to launch profiles. For example running the
-below curl command will launch a profile in gNBSim
+gNBSim can process HTTP Requests to launch profiles. 
+
+HTTP API to create new profile. Below configuration enables http server in gNBSim.
+
+
+      config:
+        gnbsim:
+          httpServer:
+            enable: true #enable httpServer in gnbsim
+            port: 6000
+
+Example to use gNBSim can be found `script <https://github.com/omec-project/gnbsim/blob/main/scripts/create-new-profile.sh>`_
+For example running the below curl command will launch a profile in gNBSim
```

Comment:
Please check the output of these lines. It looks like the output is not properly rendered.

Reply:
updated. Check once.